public/js/templates/ideaListItem.tpl

```diff
@@ -0,0 +1,6 @@
+<a class="ideaLink" href="/ideas/<%= name %>">
```

Comment:
you don't have a `name` property in your model attributes, so this ends up empty and clicking on a single idea doesn't route property